include/swift/AST/Module.h

```diff
@@ -264,7 +264,6 @@ class ModuleDecl : public DeclContext, public TypeDecl {
     return { Files.begin(), Files.size() };
   }
 
-  bool isClangModule() const;
```

Comment:
Sorry for being unclear, I actually liked that you added `isClangModule()` method, I was mainly suggesting to look into implementing this in terms of calling `findUnderlyingModule()`

Reply:
No worries! I've misread it